.travis.yml

```diff
@@ -0,0 +1,41 @@
+dist: trusty
```

Comment:
The following comment is related to the commit message:

Please replace the commit message by the message in the "original" commit https://github.com/AVnu/OpenAvnu/commit/76c6e71e2a56ba19b4527abebc24508d3e0bce76 in order to preserve the history.

Reply:
Done.